### PR TITLE
API-9670 add description for auto_accesses_updated

### PR DESCRIPTION
**Pull request by MaciejSzaflik**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/API-9264)

# Description

We're changing how webhooks and pushes for modification of auto access rules work.
Instead of multiple pushes in 3.5, we're introducing a new push/webhook - auto_accesses_updated that aggregates all changes done in single action as a single push/webhook instead of old behavior, which resulted in multiple pushes from one action.

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/management/webhooks/v3.5/index.mdx [hunk 1]
```diff
@@ -1493,55 +1493,79 @@ Informs that an agent's account was deleted.
...
 <Text>
 
-### `auto_access_added`
+### `auto_accesses_updated`
 
-Informs that a new auto access rule was added. The payload contains the full state of the auto access rule data structure, including empty fields.
+Informs that auto access rules were modified. The payload contains an array of all changes done as result of adding, updating, or deleting auto access rule.
+The resulting payload can contain multiple objects depending on the action:
+- for added auto access rule, the payload contains the complete state of the auto access rule data structure, including empty fields.
+- for updated auto access rule, contains only the updated properties
```
Comment (kamilatokarska) [inline src/pages/management/webhooks/v3.5/index.mdx:1501]:
```suggestion
- for updated auto access rule, the payload contains only the updated properties.
```

File: src/pages/management/webhooks/v3.5/index.mdx [hunk 2]
```diff
@@ -1493,55 +1493,79 @@ Informs that an agent's account was deleted.
 
 <Text>
 
-### `auto_access_added`
+### `auto_accesses_updated`
 
-Informs that a new auto access rule was added. The payload contains the full state of the auto access rule data structure, including empty fields.
+Informs that auto access rules were modified. The payload contains an array of all changes done as result of adding, updating, or deleting auto access rule.
+The resulting payload can contain multiple objects depending on the action:
+- for added auto access rule, the payload contains the complete state of the auto access rule data structure, including empty fields.
```
Comment (kamilatokarska) [inline src/pages/management/webhooks/v3.5/index.mdx:1500]:
```suggestion
- for the added auto access rule, the payload contains the complete state of the auto access rule data structure, including empty fields.
```

File: src/pages/management/webhooks/v3.5/index.mdx [hunk 3]
```diff
@@ -1493,55 +1493,79 @@ Informs that an agent's account was deleted.
...
+- for updated auto access rule, contains only the updated properties
+- for deleted auto access rule, confirms deletion
```
Comment (kamilatokarska) [inline src/pages/management/webhooks/v3.5/index.mdx:1502]:
```suggestion
- for the deleted auto access rule, the payload confirms deletion.
```

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx [hunk 4]
```diff
@@ -887,70 +887,102 @@ Informs that an agent's account was deleted.
 
 </CodeResponse>
 
-### `auto_access_added`
+### `auto_accesses_updated`
 
-Informs that a new [auto access](/management/configuration-api/v3.5/#auto-access) rule was added. The payload contains the full state of the auto access rule data structure, including empty fields.
+Informs that auto access rules were modified. The payload contains an array of all changes done as result of adding, updating, or deleting auto access rule.
+The resulting payload can contain multiple objects depending on the action:
+- for added [auto access](/management/configuration-api/v3.5/#auto-access) rule, the payload contains the complete state of the auto access rule data structure, including empty fields.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx:894]:
```suggestion
- for the added [auto access](/management/configuration-api/v3.5/#auto-access) rule, the payload contains the complete state of the auto access rule data structure, including empty fields.
```

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx [hunk 5]
```diff
@@ -887,70 +887,102 @@ Informs that an agent's account was deleted.
...
+- for updated [auto access](/management/configuration-api/v3.5/#auto-access) rule, contains only the updated properties
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx:895]:
```suggestion
- for the updated [auto access](/management/configuration-api/v3.5/#auto-access) rule, the payload contains only the updated properties.
```

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx [hunk 6]
```diff
@@ -887,70 +887,102 @@ Informs that an agent's account was deleted.
...
+- for deleted [auto access](/management/configuration-api/v3.5/#auto-access) rule, confirms deletion
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx:896]:
```suggestion
- for  the deleted [auto access](/management/configuration-api/v3.5/#auto-access) rule, the payload confirms the deletion.
```

**Review by kamilatokarska — COMMENTED** [review]
Hey :) A few lg things to correct.

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx (hunk 6 above)
Comment (MaciejSzaflik) [reply src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx]: applied

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx (hunk 5 above)
Comment (MaciejSzaflik) [reply src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx]: applied

File: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx (hunk 4 above)
Comment (MaciejSzaflik) [reply src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx]: applied

File: src/pages/management/webhooks/v3.5/index.mdx (hunk 3 above)
Comment (MaciejSzaflik) [reply src/pages/management/webhooks/v3.5/index.mdx]: applied

File: src/pages/management/webhooks/v3.5/index.mdx (hunk 2 above)
Comment (MaciejSzaflik) [reply src/pages/management/webhooks/v3.5/index.mdx]: applied

File: src/pages/management/webhooks/v3.5/index.mdx (hunk 1 above)
Comment (MaciejSzaflik) [reply src/pages/management/webhooks/v3.5/index.mdx]: applied

**Review by kamilatokarska — APPROVED** [review]
:) Please let me know when I can publish it

**Pull request merged**